src/app/shared/palette/order-summary/order-summary.model.ts

```diff
@@ -0,0 +1,7 @@
+import { FeeValue } from './fee-value.model';
+
+export class OrderSummary {
+    PaymentReference: string;
```

Comment:
how come fields are upper case?

Reply:
This is the case details data contract we have with services.